<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DreamDesktopConfig</class>
 <widget class="QWidget" name="DreamDesktopConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>327</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_0">
     <property name="text">
      <string>Framerate:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="m_fpsRate">
     <item>
      <property name="text">
       <string>Autodetect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30 fps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25 fps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20 fps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15 fps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 fps</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>Scale Filter:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="m_swsFilter">
     <item>
      <property name="text">
       <string>Fast Bilinear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bilinear</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bicubic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Point</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Area</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bicublin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gauss</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sinc</string>
      </property>
      <property name="text">
       <string>Lanczos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spline</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="KListWidget" name="m_videoList">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>220</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::Batched</enum>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="gridSize">
      <size>
       <width>150</width>
       <height>130</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>139</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="KPushButton" name="m_addVideo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Add Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="m_removeVideo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Remove Video</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
